admin/google_search_console/class-gsc-category-filters.php

```diff
@@ -152,19 +154,24 @@ private function create_view_link( $category, $count ) {
 			$class .= ' current';
 		}
 
-		$title = '';
+		$help_button = $help_panel = '';
+		$help_button_text = $this->filter_values[ $category ]['help-button'] !== '' ? $this->filter_values[ $category ]['help-button'] : __( 'Help', 'wordpress-seo' );
 		if ( $this->filter_values[ $category ]['description'] !== '' ) {
-			$title = " title='" . esc_attr( $this->filter_values[ $category ]['description'] ) . "'";
+			$help_button = '<button type="button" class="yoast_help yoast-help-button dashicons" id="' . esc_attr( $category . '-help-toggle' ) .
```

Comment:
I see code to create a help button three times in this PR, could you abstract this?

A simple help button class would suffice:
```php
class WPSEO_Admin_Help_Button {
    // (...)

    public function get_button_html() {
        // (...)
    }

    public function get_panel_html() {
        // (...)
    }
}
```

`Help_Button` might not be the best name for this, do you have a better one?